src/v3/plugins/github/relationalView.js

```diff
@@ -28,6 +29,8 @@ export class RelationalView {
   _comments: Map<N.RawAddress, CommentEntry>;
   _reviews: Map<N.RawAddress, ReviewEntry>;
   _userlikes: Map<N.RawAddress, UserlikeEntry>;
+  _mapReferences: Map<N.RawAddress, N.ReferentAddress[]>;
```

Comment:
It shouldn't be `references` because it's a private helper property. It can't be `_references` because that is the name of the helper function for finding references for an entity.

Reply:
Right, `_references` is what I meant. That makes sense. Thanks.